src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import cn from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people : Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  const columnNames = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
```

Comment:
It is better to initialize this array (`columnNames`) outside the component, so that it is not created every time during a new rendering.